src/assets/scripts/game/GameController.js

```diff
@@ -25,6 +47,7 @@ export default class GameController {
         this.game.frequency = 1;
         this.game.time = 0;
         this.game.delta = 0;
+        this.game.events = {};
```

Comment:
As we go through these files more I would love to update class properties to be initialized correctly.  There are _so_ many properties that are initialized as `null`.  Although that may be correct for classes, it is very incorrect for strings and arrays.

```javascript
const boolean = false;
const number = -1;
const string = '';
const array = [];
const object = {};
const class = null;
```
By initializing a property as invalid you do two things; you inform other developers about the data type and you immediately have and falsey type value for properties that you can check against.  There's more to it than that, but we can talk more about that some other time if you're curious.

You did good here and I'd like to see this pattern continue. 👍